javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -148,9 +148,7 @@ private TreeUtils() {
             TREEMAKER_SELECT =
                     TreeMaker.class.getMethod("Select", JCExpression.class, Symbol.class);
         } catch (NoSuchMethodException e) {
-            Error err = new AssertionError("Unexpected error in TreeUtils static initializer");
-            err.initCause(e);
-            throw err;
+            throw (Error) new AssertionError("Unexpected error in TreeUtils static initializer", e);
```

Comment:
```suggestion
            throw new AssertionError("Unexpected error in TreeUtils static initializer", e);
```

That upcast seems unnecessary. Is your IDE not complaining about that?

Reply:
Just checked it did not complain about the cast.

Reply:
Do you agree with removing the cast? Or do you see some benefit in keeping it?

Reply:
Sure, remove it is better for clean code.